pages/agent/v3/help/_lock_do.md

```diff
@@ -17,41 +17,45 @@ script.
 `buildkite-agent lock do [key]`
 
 ### Description
-Begins a do-once lock. Do-once can be used by multiple processes to 
+Begins a do-once lock. Do-once can be used by multiple processes to
 wait for completion of some shared work, where only one process should do
-the work. 
+the work.
 
-Note that this subcommand is only available when an agent has been started with
-the ′agent-api′ experiment enabled.
+Note that this subcommand is only available when an agent has been started
+with the `agent-api` experiment enabled.
 
 `lock do` will do one of two things:
 
-- Print `do`. The calling process should proceed to do the work and then
-call ′lock done′.
-- Wait until the work is marked as done (with ′lock done′) and print `done`.
+- Print &#39;do&#39;. The calling process should proceed to do the work and then
+call `lock done`.
+- Wait until the work is marked as done (with `lock done`) and print &#39;done&#39;.
 
-If `lock do` prints `done` immediately, the work was already done.
+If `lock do` prints &#39;done&#39; immediately, the work was already done.
 
 ### Examples
 
 ```shell
 #!/bin/bash
-if [ $(buildkite-agent lock do llama) = 'do' ] ; then
-setup_code()
-buildkite-agent lock done llama
+if [[ $(buildkite-agent lock do llama) == 'do' ]]; then
+  # your critical section here...
+  buildkite-agent lock done llama
 fi
 ```
 
-
 ### Options
 
 <!-- vale off -->
 
 <table class="Docs__attribute__table">
+<tr id="lock-wait-timeout"><th><code>--lock-wait-timeout value</code> <a class="Docs__attribute__link" href="#lock-wait-timeout">#</a></th><td><p>If specified, sets a maximum duration to wait for a lock before giving up (default: 0s)<br /><strong>Environment variable</strong>: <code>$BUILDKITE_LOCK_WAIT_TIMEOUT</code></p></td></tr>
```

Comment:
Same thing with "If specified" but I assume that's a change in the agent and it's not new in this change.

Reply:
I changed it.